exercises/hardware_access_control/README.md

```diff
@@ -107,7 +108,7 @@ Luckily the linker has all the information needed to check which compartments ca
 It outputs this information in a JSON report with the rest of the build artefacts.
 To automate checking this report, we can use `cheriot-audit` which should already be in your path.
 
-`cheriot-audit` allows you to query the JSON report and assert certain rules are followed.
+CHERIoT Audit allows you to query the JSON report and assert certain rules are followed.
```

Comment:
If changing here, we should probably change the sentence above so the tool is referenced in the same way.

Note: `cheriot-audit` is how it's referenced in the body of it's introductory blog post.

Reply:
Thanks I've changed it in a few more places.